README.md

```diff
@@ -29,21 +29,24 @@ This package provides a Nuke plugin that creates jobs for AWS Deadline Cloud usi
 To install the submitter manually, you can use pip.
 
 ```sh
-$ pip install deadline-cloud-for-nuke -t target-directory
+pip install deadline-cloud-for-nuke -t ~/deadline-cloud-for-nuke
 ```
+Please note that target directory is `~/deadline-cloud-for-nuke` in the example, but you can choose any other directorys that you'd prefer.
 
 For Windows,
 ```sh
-$ set NUKE_PATH=%NUKE_PATH%;<target-directory>/deadline/nuke_submitter
+set NUKE_PATH=%NUKE_PATH%;~/deadline-cloud-for-nuke/deadline/nuke_submitter
```

Comment:
~ is not a valid path on Windows when using the command prompt so I'm not sure that this would work for Windows users with Nuke. Did you manage to test the instructions on a Windows machine and have it work?

Reply:
Yeah, updated this and verified in Windows machine whether it updates environment variable as I wanted to.